Create 2025-05-14-domain-category-improvements.mdx (#22417)
cloudflare/cloudflare-docs: src/content/changelog/gateway/2025-05-14-domain-category-improvements.mdx

---
title: Domain Categories improvements

> description: We've added 7 new Domain Categories and renamed 3 Domain Categories, to better reflect online content and enable DNS filtering

Reviewer: ```suggestion
description: We have added 7 new Domain Categories and renamed 3 Domain Categories, to better reflect online content and enable DNS filtering
```

date: 2025-05-14T11:00:00Z
---

> **New Categories Added**

Reviewer: ```suggestion
**New categories added**
```

| Parent ID | Parent Name | Category ID | Category Name |
| --- | --- | --- | --- |
| 1 | Ads | 66 | Advertisements |
| 3 | Business & Economy | 185 | Personal Finance |
| 3 | Business & Economy | 186 | Brokerage & Investing |
| 21 | Security Threats | 187 | Compromised Domain |
| 21 | Security Threats | 188 | Potentially Unwanted Software |
| 6 | Education | 189 | Reference |
| 9 | Government & Politics | 190 | Charity and Non-profit |

> **Changes to Existing Categories**

Reviewer: ```suggestion
**Changes to existing categories**
```

| Original Name | New Name |
| --- | --- |
| Religion | Religion & Spirituality |
| Government | Government/Legal |
| Redirect | URL Alias/Redirect |

> Learn more in the [Gateway domain categories developer docs](/cloudflare-one/policies/gateway/domain-categories/) 

Reviewer: ```suggestion
Refer to [Gateway domain categories](/cloudflare-one/policies/gateway/domain-categories/) to learn more.
```
